suites/nautilus/rgw/tier_1_rgw_multisite_primary_to_secondary.yaml

```diff
@@ -0,0 +1,375 @@
+"""
+Below are the multi-site test scenarios run on the master and verified the sync on the slave
+
+where :
+ceph-rgw1 is master/primary site
+ceph-rgw2 is slave/secondary site
+
+"""
```

Comment:
The syntax for comments is different for a yaml file. We would need to start the each line with `#`

Reply:
@viduship see this [yaml_comments](https://www.tutorialspoint.com/yaml/yaml_comments.htm) . if you had used an editor, the error would have come up instantly :)

Reply:
Done